src/types/libdef/npm-custom/@testing-library/react_v11.x.x.js

```diff
@@ -234,15 +439,15 @@ declare module '@testing-library/react' {
         | Array<HTMLElement | DocumentFragment>,
       maxLength?: number
     ) => void,
-    rerender: (ui: React$Element<*>) => void,
+    rerender: (ui: React$Element<any>) => void,
```

Comment:
note: this is the only other thing I changed in from the react_v11 libdef. Other `*` => `any` changes came from the libdef itself.